apps/condo/domains/common/utils/text.js

```diff
@@ -0,0 +1,44 @@
+function removeSpacesInsideQuotes (s) {
+    const nestedQuotes = []
+    let res = [...s]
+    for (let i = 0; i < res.length; i++) {
+        const char = res[i]
+        if (char === '"') {
+            if (nestedQuotes[nestedQuotes.length - 1] === char) {
+                nestedQuotes.pop()
+                if (res[i - 1] === ' ') res[i - 1] = ''
+            } else {
+                nestedQuotes.push(char)
+                if (res[i + 1] === ' ') res[i + 1] = ''
+            }
+        }
+    }
+    return res.join('')
+}
```

Comment:
Looks tricky ;)
Can it be replaces with regular expressions?
Also text can have single quotes.
Just as adea — `/['"](\ +).*['"]`

Reply:
Single quotes can be used as apostrophes. Then everything can break

Reply:
Yeah, it will be tricky.
Ok.